Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,591 @@
+
+### Setup our Game Server
+
+We will be running our game on an AWS EC2 Instance running Windows Server, these instructions also work with a Linux server using minor modifications.
+
+First create a key pair in the AWS console for remote access to the machine:
+
+![access_key_quickstart](access_key_quickstart.png)
+
+Next, configure terraform to provision an ec2 instance where we will run the server, this should be in `ec2/main.tf`
+
+```
+resource aws_instance "server_instance" {
+    ami = "ami-0f496107db66676ff"
+    instance_type = "t2.medium"
+    key_name = "windows-server"
+
+    root_block_device {
+        volume_size = 100
+    }
+
+    user_data = <<-EOF
+<powershell>
+# Open port 7777 UDP for the game server
+New-NetFirewallRule -Direction Inbound -LocalPort 7777 -Protocol UDP -Action Allow -DisplayName gameserver
+EOF
+
+    vpc_security_group_ids = [var.security_group_id]
+    subnet_id = var.subnets[0].id
+
+    tags = {
+        Name = "${var.workspace_prepend}Windows Server"
+    }
+}
+```
+
+Additionally, we should add additional exposed ports to the security group:
+
+> For the purpose of this tutorial, we can add these to the existing security group, but in production it is a better choice to have 
+> separate security groups for your game servers and to not expose RDP
+
+```
+    ingress {
+        description = "Game Server Traffic"
+        from_port   = 7777
+        to_port     = 7777
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"] 
+    }
+
+    ingress {
+        description = "RDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        description = "RDP UDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+```
+
+Next plan and apply your infrastructure
+
+```
+terraform plan -var-file="vars.tfvar" 
+terraform apply -var-file="vars.tfvar"
+```
+
+Once Terraform finishes, you can go to the AWS EC2 Console, click your instance, and click "Connect to instance." Under the RDP tab, download the
+Remote Desktop File. Then click "Get Password" and paste in the private key from the keypair you created previously.
+
+At this point you can connect to the Windows server using RDP by opening the file and logging into the machine with the "Administrator" account and password
+from the AWS console.
+
+At this point you should download and prepare to run your game on the server. Instructions for the Lyra Demo are below.
+
+<procedure title="Lyra Game Server Installation" id="lyra_setup" collapsible="true">
+    <p>These instructions use the <a href="https://github.com/catenatools/catena-lyra-demo">catena-lyra-demo</a> server.</p>
+    <step>Download the server onto the machine. The download can be found <a href="https://catena-public-content.S3.amazonaws.com/WindowsServer.zip">here.</a></step>
```

Comment:
Last big comment that just occurred to me. We may want to provide some instructions for copying local files to their remote desktop/remote machine if they're not going to be using Lyra.

Linux: `scp`
Windows: https://serverfault.com/questions/150369/how-to-copy-files-to-amazon-ec2-windows-instance-from-my-local-machine